### DISCUSSION TITLE
震惊，《原神》纳塔交响音乐现场评论区已失控，玩家们要冲160w评论？

### DISCUSSION THREAD

**Discussion by x97-1** [share121/inter-knot#3422] [其他]
###  现在已经80w了，还是在ac站严重限流的情况下。
<img width="378" alt="截屏2024-08-21 22 20 07" src="https://github.com/user-attachments/assets/01917544-9ebe-40b6-9e64-4a9e159c10ba">

> 当然我也冲了，ac站现在越来越不干人事了。

> ### 只要omo一声令下，我将立马拥护它为新的b站<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by hk0852** [comment]
九月八日考古，已经800万播放，2298107评论了，ac站终究是菜就多练<img src="https://img-static.mihoyo.com/communityweb/upload/677e18698f28c655cd39bfb7f16a86f2.png" alt="卡芙卡 怒" width="75" >

**Comment by 11-billy** [comment]
ac站也就那样了

**Comment by HoshimiMiyabi120** [comment]
鸡爪流神力
